mmv1/templates/tgc/resource_converters.go.erb

```diff
@@ -51,6 +51,7 @@ func ResourceConverters() map[string][]cai.ResourceConverter {
 		"google_compute_ssl_policy":                       {compute.ResourceConverterComputeSslPolicy()},
 		"google_compute_target_https_proxy":               {compute.ResourceConverterComputeTargetHttpsProxy()},
 		"google_compute_target_ssl_proxy":                 {compute.ResourceConverterComputeTargetSslProxy()},
+		"google_dataproc_autoscaling_policy":			   {dataproc.ResourceConverterDataprocAutoscalingPolicy()},
```

Comment:
Can you align this using spaces? It currently mixes spaces + tabs

Reply:
sure (: +1: